services/QuillLMS/engines/evidence/lib/evidence/check.rb

```diff
@@ -11,8 +11,11 @@ class NoMatchedFeedbackTypesError < StandardError; end
       Plagiarism
     ]
 
-    AI_CHECK_MAPPING = Evidence::Activity::AI_TYPES
-      .index_with { |type| "Evidence::Check::#{type}".constantize }
+    AI_CHECK_MAPPING = {
+      Evidence::Activity::AUTO_ML => Evidence::Check::AutoML,
+      Evidence::Activity::GEN_AI => Evidence::Check::GenAi,
```

Comment:
`GEN_AI = 'GenAI'` which is then used in `{ |type| "Evidence::Check::#{type}".constantize }` but this would break since the constant is now `GenAi`.  However, we can't change to ``GEN_AI = 'GenAi' since that constant is used in many places in the frontend.